static/js/util/profile_edit.js

```diff
@@ -50,7 +50,7 @@ export function boundSelectField(keySet, label, options) {
   } = this.props;
   let onChange = value => {
     let clone = _.cloneDeep(profile);
-    _.set(clone, keySet, value? value.value : null);
+    _.set(clone, keySet, value? value.value : '');
```

Comment:
was this causing a console warning? I know we had an issue with the `TextField` complaining when `this.props.value == undefined`.